homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -42,6 +41,17 @@ def __init__(self, account, vehicle, attribute: str, sensor_name):
             DOOR_LOCK_STATE in self._vehicle.available_attributes
         )
 
+    @property
+    def device_info(self) -> dict:
```

Comment:
Can we find a way of sharing this function between entities ?

Reply:
It definitely can when using something like a `BMWBaseEntity`.
I left that out as I have another PR with `DataUpdateCoordinator` where this is part and I didn't want to increase the size of this PR even more.